sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -7658,6 +7668,19 @@ def test_Str():
     from sympy.core.symbol import Str
     assert pretty(Str('x')) == 'x'
 
+
+
+def test_symbolic_probability():
+    mu = symbols("mu")
+    sigma = symbols("sigma", positive=True)
+    X = Normal("X", mu, sigma)
+    assert pretty(Expectation(X)) == r'E[X]'
+    assert pretty(Variance(X)) == r'var(X)'
+    assert pretty(Probability(X > 0)) == r'P(X > 0)'
+    Y = Normal("Y", mu, sigma)
+    assert pretty(Covariance(X, Y)) == 'cov(X, Y)'
```

Comment:
I would probably capitalise Var and Cov

Reply:
English Wikipedia writes cov, but I see many other sources using Cov (including Swedish Wikipedia). Easily fixed!

Reply:
This is changed in #22661 (this PR is built on top of that).

Reply:
Since #22661 is merged current master shows:
```python
In [5]: pretty(Covariance(X, Y))
Out[5]: 'Cov(X, Y)'
```
Is the intention to change that here?

Reply:
I was under the impression that the first commit, which is merged and later overridden elsewhere, would be removed from this PR. But I must have missed how it works (or at least displays).

I'll rebase this and remove that commit.

Reply:
Hmm, when rebasing that commit was automatically removed anyway. So not sure what would have happened if this was merged as is (and will not find out...).